indico/modules/events/registration/client/js/form/FormItem.jsx

```diff
@@ -97,6 +97,7 @@ export default function FormItem({
   }
 
   const required = meta.alwaysRequired || isRequired;
+  const managementRequired = !isManagement && required;
```

Comment:
When testing I didn't notice any problems since we always preselect the "no accommodation" option, but if the accommodation field does not have that option (you can disable it if the user MUST check one), then it currently fails silently regardless of the override toggle.

Reply:
For the sake of getting this PR in I'll go with the fix you suggested, but I do feel that event managers need to be able to make changes to a registration without affecting any required fields. 

If the event manager is required to fill out the accommodation field, we're missing the state "registrant has not filled this out yet", which is available for all other fields. 

If the manager is required to select "No accommodation", this will make it harder to differentiate between attendees that have deliberately decided against accommodation vs those who need more nagging to fill out the registration field.

Reply:
True, that's a good point I didn't think about. I'll have a look before merging this if I can find a nice way to solve it.